dockerfiles/base/scripts/base/commands/cmd_debug.sh

```diff
@@ -19,7 +19,12 @@ cmd_debug() {
   info "${CHE_PRODUCT_NAME}_VERSION           = ${CHE_VERSION}"
   info "${CHE_PRODUCT_NAME}_INSTANCE          = ${CHE_HOST_INSTANCE}"
   info "${CHE_PRODUCT_NAME}_CONFIG            = ${CHE_HOST_CONFIG}"
-  info "${CHE_PRODUCT_NAME}_HOST              = ${CHE_HOST}"
+  local CHE_HOST_LOCAL=${CHE_HOST}
+  if is_initialized; then
+    CHE_HOST_LOCAL=$(docker_run --env-file="${REFERENCE_CONTAINER_ENVIRONMENT_FILE}" \
```

Comment:
Maybe create function that retrieves variable by name? It is possible that other variables in different places will be also needed at some point